### ISSUE TITLE
tgview shows uses links as undirected

### ISSUE THREAD

**Issue by kohlhase** [slatex/lmhtools#3]
I wonder why.

**Comment by jfschaefer** [comment]
The edges have the style "uses", which seems to be undefined in TGView, so apparently it falls back to a default style for edges.

The edge styles are defined [here](https://github.com/UniFormal/TGView/blob/a6aa4db9e27ad6b25825af147e9a4db15f1aa171/src/Configuration.ts#L220). We should probably think about what uses edges should look like and add a corresponding entry.

**Comment by kohlhase** [comment]
yes, that is a good plan. how about making them magenta or cyan?

**Comment by kohlhase** [comment]
cyan is better, could you do that?

**Comment by jfschaefer** [comment]
Yes. I've made a pull request for it: [https://github.com/UniFormal/TGView/pull/60](https://github.com/UniFormal/TGView/pull/60)
